doc/tutorials/install-webui.md

```diff
@@ -165,6 +165,44 @@ will now return the value of the specified key `user:/test`, which is stored in
 
 <!-- prettier-ignore-end -->
 
+The command
+
+```sh
+curl -X POST -H "Content-Type: application/json" -d '{"metaSet": [{"key": "metakey1", "value": "value1"},{"key": "metakey2", "value": "value2"}]}' http://localhost:33333/kdbMetaBulk/user:/test
```

Comment:
Why is it called `metaSet` and not just `meta`?

Reply:
No specific reason :smile:. Will change it to `meta`.